pkgs/by-name/wp/wprs/package.nix

```diff
@@ -41,6 +50,13 @@ rustPlatform.buildRustPackage rec {
 
   preFixup = ''
     cp  wprs "$out/bin/wprs"
+
+    # wprs is a raw python script that needs a module.
+    wrapPythonProgramsIn "$out/bin" "${pythonPath}"
+  '';
+
+  passthru.tests.sanity = runCommand "wprs-sanity" { nativeBuildInputs = [ wprs ]; } ''
+    ${wprs}/bin/wprs -h > /dev/null && touch $out
```

Comment:
Would be possible to run `testers.testVersion`?

Reply:
Yeah I would prefer `testers.testVersion` but the binary doesn't have a version arg.